### PR TITLE
go: update to 1.8.3.

### PR DESCRIPTION
**Pull request by ebfe**
As always manual rebuild of host go on the aarch64 builders required before pushing the revbumps.

### CODE REVIEW

**Comment by ebfe** [comment]
updated to go 1.8.3

File: srcpkgs/flannel/template [hunk 1]
```diff
@@ -10,7 +10,7 @@ license="Apache-2.0"
 go_import_path="github.com/coreos/flannel"
 homepage="https://$go_import_path"
 distfiles="$homepage/archive/v${version}.tar.gz"
-checksum=f8b837bc042c917cd37cfe131132ba1fde5268fadb7632da9c16f57d0905161b
+checksum=3c3148eac3537fba6c5377922366ed36b1415d20111434172c5c42fec74db366
```
Comment (Gottox) [inline srcpkgs/flannel/template]: Any idea what has changed here?

Comment (thypon) [inline srcpkgs/flannel/template]: They probably force pushed a tag to a different revision.

Comment (ebfe) [inline srcpkgs/flannel/template]: the only diff is in a version string of a dependency.

Comment (Gottox) [inline srcpkgs/flannel/template]: 🙄 when will they learn.

**Pull request merged**